keyboards/cipulot/ec_virgo/info.json

```
@@ -0,0 +1,130 @@
+{
+    "manufacturer": "Cipulot",
+    "keyboard_name": "Virgo EC",
+    "maintainer": "Cipulot",
+    "bootloader": "stm32-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096
+        }
+    },
+    "features": {
+        "bootmagic": false,
```

This is preferred to be enabled, unless there's a reason not to.
```suggestion
        "bootmagic": true,
```

---

Bootmagic breaks some EEPROM and scan behavior as discussed in a chat in the QMK developer discord. I even later bypass it with a dummy call to be sure that it isn't re-enabled by via.
This should remain disabled